lib/Zonemaster/Backend/DB.pm

```diff
@@ -51,22 +51,25 @@ sub get_db_class {
 sub user_exists {
     my ( $self, $user ) = @_;
 
-    die "username not provided to the method user_exists\n" unless ( $user );
+    die Zonemaster::Backend::Error::Internal->new( reason => "username not provided to the method user_exists")
+    unless ( $user );
```

Comment:
Could add an indentation here before the `unless` to make it easier to read?

Reply:
Done